shopfloor_workstation/tests/test_shopfloor_workstation.py

```diff
@@ -60,3 +64,20 @@ def test_workstation_set_default_found(self):
             },
         )
         self.assertEqual(self.env.user.printing_printer_id, self.printer1)
+
+    def test_workstation_constraints(self):
+        with self.assertRaises(IntegrityError), self.env.cr.savepoint(), mute_logger(
```

Comment:
It would be better to test w/ assertRaisesRegex to check the specific msg.
Not a blocker tho.